x/hold/hold_test.go

```diff
@@ -0,0 +1,70 @@
+package hold
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/provenance-io/provenance/testutil/assertions"
+)
+
+func TestAccountHold_Validate(t *testing.T) {
+	cz := func(coins string) sdk.Coins {
+		rv, err := sdk.ParseCoinsNormalized(coins)
+		require.NoError(t, err, "ParseCoinsNormalized(%q)", coins)
+		return rv
+	}
+	c := func(amount int64, denom string) sdk.Coin {
```

Comment:
This looks the same as sdk.NewInt64Coin().

Reply:
Not quite. `NewInt64Coin` runs validation and panics if there's something wrong.